packages/server/api/src/app/flows/flow/flow-version.controller.ts

```diff
@@ -189,6 +191,52 @@ export const flowVersionController: FastifyPluginAsyncTypebox = async (
       });
     },
   );
+
+  fastify.post(
```

Comment:
@MarceloRGonc why did you want this under this route? 
I think it's a bit weird, shouldn't this be a separate route/entity, of "runs"? 
we're calling stepRunService, which is another clue that this doesn't belong here, it's a separate entity.